ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -811,6 +811,7 @@ tests:
       DEVSCRIPTS_CONFIG: |
         AGENT_E2E_TEST_SCENARIO=COMPACT_IPV4
         AGENT_PLATFORM_TYPE=none
+        NETWORK_TYPE=OVNKubernetes
```

Comment:
In dev-scripts, OVN is set with SNO so not necessary to set here:
https://github.com/openshift-metal3/dev-scripts/blob/master/common.sh#L436

Reply:
You're right, nvr mind. I thought we automatically set for `none` too but looks like we don't.